ledger/sandbox/src/main/scala/com/digitalasset/platform/index/TransactionConversion.scala

```diff
@@ -143,7 +154,8 @@ object TransactionConversion {
     val (replaced, roots) =
       tx.roots.foldLeft((false, IndexedSeq.empty[EventId])) {
         case ((replaced, roots), eventId) =>
-          if (disclosure.contains(eventId)) (replaced, roots :+ eventId)
+          if (isCreateOrExercise(tx.nodes(eventId)) && disclosure.contains(eventId))
```

Comment:
Can we add some curly braces here please?

Reply:
Thanks. Addressed in 7c1c9299b.